lib/Doctrine/Common/Annotations/TokenParser.php

```diff
@@ -48,6 +48,17 @@ public function __construct($contents)
         token_get_all("<?php\n/**\n *\n */");
 
         $this->numTokens = count($this->tokens);
+
+        // Compatibility defines for PHP < 8.0.
```

Comment:
Do we even need those now that we have separate methods ?

Reply:
I did use it in https://github.com/doctrine/annotations/pull/344/files#diff-0eaa10dd1d548d163b15102395644457R213
I can split them also in 2 methods, if you want :)

Reply:
Also mocking fails, when I remove it

Reply:
well, that other line of code could be changed to `(\PHP_VERSION_ID >=80000 && $token[0] === T_NAME_QUALIFIED)`, which will avoid having 2 methods and won't have any overhead (as the version check can be resolved at compile time)

Reply:
Changed. Thanks :)